packages/public/server/src/module/Subject/config/module.config.php

```diff
@@ -108,12 +108,12 @@
                 ],
                 'child_routes' => [
                     'entity' => [
-                        'type'    => 'segment',
+                        'type'    => 'literal',
                         'options' => [
-                            'route'    => '/entity/:action',
+                            'route'    => '/entity/trash-bin',
```

Comment:
Note: This would change the current URL of the trash-bin...

Reply:
https://de.serlo.org/mathe/entity/trash-bin also works atm :-)